src/Cake.Svn/Info/SvnInfoResult.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using Cake.Svn.Internal.Extensions;
+
+namespace Cake.Svn.Info
+{
+    /// <summary>
+    /// Result for <see cref="SvnInfo"/>.
+    /// </summary>
+    public sealed class SvnInfoResult
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="SvnInfoResult"/> class.
+        /// </summary>
+        public SvnInfoResult(
+            Guid repositoryId, 
+            Uri repositoryRoot, 
+            string lastChangedAuthor,
+            long revision,
+            Uri uri,
+            string path,
+            string fullPath,
+            SvnKind nodeKind)
+        {
+            repositoryRoot.NotNull(nameof(repositoryRoot));
+            lastChangedAuthor.NotNull(nameof(lastChangedAuthor));
+            uri.NotNull(nameof(uri));
+            path.NotNull(nameof(path));
+            fullPath.NotNull(nameof(fullPath));
+
+            RepositoryId = repositoryId;
+            RepositoryRoot = repositoryRoot;
+            LastChangedAuthor = lastChangedAuthor;
+            Revision = revision;
+            Uri = uri;
+            Path = path;
+            FullPath = fullPath;
+            NodeKind = nodeKind;
+        }
+
+        /// <summary>
+        /// Gets the uuid of the repository (if available). Otherwise Guid.Empty.
+        /// </summary>
+        public Guid RepositoryId { get; }
+
+        /// <summary>
+        ///  Gets the repository root Uri; ending in a '/'.
+        /// </summary>
+        /// <remarks>
+        /// The unmanaged api does not add the '/' at the end, but this makes using System.Uri hard.
+        /// </remarks>
+        public Uri RepositoryRoot { get; }
+
+        /// <summary>
+        ///  Gets the author of the last revision in which node (or one of its descendants) changed.
+        /// </summary>
+        public string LastChangedAuthor { get; }
+
+        /// <summary>
+        /// Gets the queried revision.
+        /// </summary>
+        public long Revision { get; }
+
+        /// <summary>
+        /// Gets the full Uri of the node.
+        /// </summary>
+        public Uri Uri { get; }
+
+        /// <summary>
+        /// Gets the path of the file. The local path if requisting WORKING version, otherwise
+        /// the name of the file at the specified version.
+        /// </summary>
+        public string Path { get; }
+
+        /// <summary>
+        /// Gets the path the notification is about, translated via SharpSvn.SvnTools.GetNormalizedFullPath(System.String).
```

Comment:
How it is translated is an implementation detail and of no interest for the consumer of the API. Should be comented where it acually happens